docs/content/en/docs/concepts/tracing-policy/hooks.md

```diff
@@ -387,7 +387,7 @@ arguments to include in the trace output. Tetragon's BPF code requires
 information about the types of arguments to properly read, print and
 filter on its arguments. This information needs to be provided by the user under the
 `args` section. For the [available
-types](https://github.com/cilium/tetragon/blob/main/pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml#L64-L88),
+types](https://github.com/cilium/tetragon/blob/main/pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml#L166-L208),
```

Comment:
```suggestion
types](https://github.com/cilium/tetragon/blob/v1.4.0/pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml#L166-L208),
```
Let's maybe point these lines to the ones of v1.4.0 so that this link is not broken in a few patches

Reply:
Thanks @MickaelFontes appreciate the docs updates. Maybe just follow up with Mahe's comments.